frontend/src/features/admin-form/settings/components/EndPageSettingsSection/EndPageSettingsInput.tsx

```diff
@@ -0,0 +1,93 @@
+import { useCallback } from 'react'
+import { useForm } from 'react-hook-form'
+import { Flex, FormControl, Stack } from '@chakra-ui/react'
+import validator from 'validator'
+
+import { FormEndPage } from '~shared/types'
+
+import Button from '~components/Button'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import FormLabel from '~components/FormControl/FormLabel'
+import Input from '~components/Input'
+import Textarea from '~components/Textarea'
+
+import { useMutateFormPage } from '~features/admin-form/common/mutations'
+
+interface EndPageSettingsInputProps {
+  endPage: FormEndPage
+}
+
+export const EndPageSettingsInput = ({
+  endPage,
+}: EndPageSettingsInputProps): JSX.Element => {
+  const { mutateFormEndPage } = useMutateFormPage()
+
+  const buttonLinkValidation = useCallback(
+    (url) =>
+      !url ||
+      validator.isURL(url, {
+        protocols: ['https'],
+        require_protocol: true,
+      }) ||
+      'Please enter a valid URL (starting with https://)',
+    [],
+  )
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormEndPage>({
+    mode: 'onChange',
+    defaultValues: endPage,
+  })
+
+  const handleUpdateEndPage = handleSubmit((endPage) =>
+    mutateFormEndPage.mutate(endPage),
+  )
+
+  return (
+    <Stack gap="2rem" paddingTop="2.5rem">
+      <FormControl isInvalid={!!errors.title}>
+        <FormLabel isRequired>Title</FormLabel>
+        <Input
+          {...register('title', {
+            required: 'Title is required for the Thank You page',
+          })}
+        />
+        <FormErrorMessage>{errors.title?.message}</FormErrorMessage>
+      </FormControl>
+      <FormControl isInvalid={!!errors.paragraph}>
+        <FormLabel isRequired>Follow-up instructions</FormLabel>
+        <Textarea
+          {...register('paragraph', {
+            required:
+              'Follow-up instructions are required for the Thank You page',
+          })}
+        />
+        <FormErrorMessage>{errors.paragraph?.message}</FormErrorMessage>
+      </FormControl>
+      <Stack
+        direction={{ base: 'column', md: 'row' }}
+        gap={{ base: '2rem', md: '1rem' }}
+      >
+        <FormControl isInvalid={!!errors.buttonText}>
+          <FormLabel isRequired>Button text</FormLabel>
+          <Input {...register('buttonText')} />
```

Comment:
might need a placeholder here too, the current one seems to be `'Submit another form'`

Reply:
I think "Submit another form" is a default value instead of a placeholder though. Currently it already defaults to "Submit another form" for the `buttonText`

Reply:
the db schema doesn't actually require a value for `buttonText`, and instead the client defaults to "Submit another form" if the value does not exist.

Reply:
Okay then I'll add the placeholder